multimodal-search-pdf/tests/requirements.txt

```diff
@@ -1,11 +0,0 @@
-git+https://github.com/jina-ai/jina.git@v1.2.0#egg=jina[hub, http]
-click==7.1.2
```

Comment:
You need this file because otherwise the CI won't run the tests (See examples/.github/ci.sh).
Please add it again and add the same dependencies as in the other requirements.txt .

Reply:
Good to know, https://github.com/jina-ai/examples/pull/629/commits/6bb2206a52013c8a6fdeefbf9a363b5fb7ab5869